installer/pkg/config-generator/generator.go

```diff
@@ -61,6 +61,73 @@ func New(cluster config.Cluster) ConfigGenerator {
 	}
 }
 
+// maoOperatorConfig contains configuration for mao managed stack
```

Comment:
"github.com/coreos/tectonic-config" will not be used for any new operators. Should we vendor in MAO's type ?

cc: @openshift/installer

Reply:
I might be misunderstanding the question, but this is kind of an MVP and the types in the machine-api-operator are likely to change significantly soon. So I think this little bit of copying / inlining is probably best for now.